.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Python CI
+
+on:
+   push:
+     branches: [ master ]
+   pull_request:
+     branches: [ master ]
+
+
+jobs:
+   run_tests:
+     name: tests
+     runs-on: ${{ matrix.os }}
+     strategy:
+       matrix:
+         os: [ubuntu-20.04]
+         python-version: ['3.5','3.8']
+         toxenv: ['django22', 'quality']
+         exclude:
+           - python-version: '3.5'
+             toxenv: 'quality'
+
+     steps:
+       - uses: actions/checkout@v2
+       - name: setup python
+         uses: actions/setup-python@v2
+         with:
+           python-version: ${{ matrix.python-version }}
+
+       - name: Install pip
+         run: pip install -r requirements/pip.txt
+
+       - name: Install Dependencies
+         run: pip install -r requirements/ci.txt
+
+       - name: Run Tests
+         env:
+           TOXENV: python${{ matrix.python-version }}-${{ matrix.toxenv }}
```

Comment:
we can just specify `TOXENV: ${{ matrix.toxenv }}`, it'll automatically use the python version currently being used by `matrix`